datasets/wmt20_mlqe_task1/wmt20_mlqe_task1.py

```diff
@@ -0,0 +1,197 @@
+# limitations under the License.
+"""WMT20 MLQE Shared task 1."""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+_CITATION = """
+Not available.
+"""
+
+_DESCRIPTION = """\
+This shared task (part of WMT20) will build on its previous editions
+to further examine automatic methods for estimating the quality
+of neural machine translation output at run-time, without relying
+on reference translations. As in previous years, we cover estimation
+at various levels. Important elements introduced this year include: a new
+task where sentences are annotated with Direct Assessment (DA)
+scores instead of labels based on post-editing; a new multilingual
+sentence-level dataset mainly from Wikipedia articles, where the
+source articles can be retrieved for document-wide context; the
+availability of NMT models to explore system-internal information for the task.
+
+Task 1 uses Wikipedia data for 6 language pairs that includes high-resource
+English--German (En-De) and English--Chinese (En-Zh), medium-resource
+Romanian--English (Ro-En) and Estonian--English (Et-En), and low-resource
+Sinhalese--English (Si-En) and Nepalese--English (Ne-En), as well as a
+dataset with a combination of Wikipedia articles and Reddit articles
+for Russian-English (En-Ru). The datasets were collected by translating
+sentences sampled from source language articles using state-of-the-art NMT
+models built using the fairseq toolkit and annotated with Direct Assessment (DA)
+scores by professional translators. Each sentence was annotated following the
+FLORES setup, which presents a form of DA, where at least three professional
+translators rate each sentence from 0-100 according to the perceived translation
+quality. DA scores are standardised using the z-score by rater. Participating systems
+are required to score sentences according to z-standardised DA scores.
+"""
+
+_HOMEPAGE = "http://www.statmt.org/wmt20/quality-estimation-task.html"
+
+_LICENSE = "Unknown"
+
+_LANGUAGE_PAIRS = [("en", "de"), ("en", "zh"), ("et", "en"), ("ne", "en"), ("ro", "en"), ("si", "en"), ("ru", "en")]
+_MAIN_URL = "https://github.com/facebookresearch/mlqe/raw/master/data"
+
+
+def inject_to_link(src_lg, tgt_lg):
+    links = {
+        "train+dev": f"{_MAIN_URL}/{src_lg}-{tgt_lg}.tar.gz",
+        "test": f"{_MAIN_URL}/{src_lg}-{tgt_lg}_test.tar.gz",
+    }
+    return links
+
+
+_URLs = {f"{src_lg}-{tgt_lg}": inject_to_link(src_lg, tgt_lg) for (src_lg, tgt_lg) in _LANGUAGE_PAIRS}
+_URLs["ru-en"] = {
+    "train+dev": "https://www.quest.dcs.shef.ac.uk/wmt20_files_qe/ru-en.tar.gz",
+    "test": "https://www.quest.dcs.shef.ac.uk/wmt20_files_qe/ru-en_test.tar.gz",
+}
+
+
+class WmtMlqeConfig(datasets.BuilderConfig):
+    def __init__(self, src_lg, tgt_lg, **kwargs):
+        super(WmtMlqeConfig, self).__init__(**kwargs)
+        self.src_lg = src_lg
+        self.tgt_lg = tgt_lg
+
+
+class WmtMlqeTask1(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Wmt20MlqeTask1(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `wmt20_mlqe_task1`